customizations/README.md

```diff
@@ -650,13 +652,38 @@ externalRedis:
 
 ### Configure external proxy
 
-CARTO stack supports working behind a HTTP/HTTPS proxy. This proxy is used to connect to external services like Google APIs, Mapbox, etc.
+CARTO self-hosted provides support for operating behind an HTTP proxy. The proxy acts as a gateway, enabling CARTO self-hosted components to establish connections with essential external services like Google APIs, Mapbox, and others.
 
-You can find customizations examples for different proxies in the [proxy](./proxy/) directory.
+> :warning: Please note that at the moment, only HTTP proxy is supported.
 
-The `externalProxy.excludedDomains` property can contain a list of all the domains that shouldn't be proxied. This is useful for example to avoid proxying the internal services like the internal Redis or Postgresql.
+You can use this [customizations](../customizations/proxy/) file as an example.
 
-> :warning: Your proxy user/password credentials will appear in clear text in the environment variables of the CARTO stack components
+The externalProxy.excludedDomains property allows you to specify a comprehensive list of domains that should not be proxied. This is useful in scenarios where you want to exclude certain services, such as internal Redis or Postgresql, from being routed through the proxy.
```

Comment:
```suggestion
The externalProxy.excludedDomains property allows you to specify a comprehensive list of domains that should not be proxied. 
```

I've removed the redis / postgresql example because they use protocols that are not supported by the proxy AFAIK (port 5432 and port 6379).